app/notifications/process_notifications.py

```diff
@@ -148,11 +150,9 @@ def send_notification_to_queue(notification, research_mode, queue=None, recipien
 
     try:
         tasks = [deliver_task.si(str(notification.id)).set(queue=queue)]
-        if (
-                recipient_id_type
-                and is_feature_enabled(FeatureFlag.CHECK_RECIPIENT_COMMUNICATION_PERMISSIONS_ENABLED)
-                and communication_item_id
-        ):
+        if (recipient_id_type and communication_item_id and
```

Comment:
Cut the parens for the if block

Reply:
The problem with this particular case if this. If we put it all on one line (no parens), the line becomes too long for our flake8 validation. I can use the `# noqa: E501` to have it not check for that if we _really_ want it to all be on one line. If we split the `if` statement across multiple lines, the parens need to stay, or that causes errors.

Reply:
I was mistaken/out of date here. 
PEP8 calls for maximum length lines to [wrap expressions in parens](https://peps.python.org/pep-0008/#maximum-line-length). 

>Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.